### PR TITLE
restarter v0.0.1: Initial release

### PR DESCRIPTION
**Pull request by xoen**
Internal API to restart applications in a given
namespace.

This API doesn't have an Ingress resource so it's
only accessible from within the k8s cluster (at `${RELEASE_NAME}.${NAMESPACE}.svc.cluster.local`).

Furthermore, the service account for this API is only able to list and
patch Deployments within the namespace where the chart/API is been
released to.

Part of ticket: https://trello.com/c/zb9HsZca/254-investigate-ways-to-trigger-apps-restart-when-data-changes

### CODE REVIEW

**Comment by xoen** [comment]
FYI: Need to do some manual end-to-end testing in `dev` but at least it deploys and it runs...

File: charts/restarter/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+The internal API will be available at {{ .Release.Name }}.{{ .Release.Namespace }}.svc.cluster.local
```
Comment (r4vi) [inline charts/restarter/templates/NOTES.txt]: nice you actually used the NOTES.txt for something useful

Comment (xoen) [reply charts/restarter/templates/NOTES.txt]: Always trying ☺️. We're users too.

Comment (xoen) [reply charts/restarter/templates/NOTES.txt]: I always try when it makes sense :) 

Good to have some meaningful information after you just installed an helm chart.

**Comment by xoen** [comment]
FYI: I've added [this `NetworkPolicy`](https://github.com/ministryofjustice/analytics-platform-helm-charts/blob/1dd9e89316c3ce9e79970e376e1146af94050a3d/charts/restarter/templates/networkpolicy.yaml) as if we deploy this in the `apps-prod` namespace only inbound traffic from internet (Ingress Controller pods) will be allowed - which is good - but also a pain :P

**Pull request merged**